app/src/hooks/cache.ts

```diff
@@ -38,6 +39,19 @@ export const useEvents = () => {
   );
 };
 
+const pastEventCache = cachedRemoteData<string, IEvent>();
+
+export const usePastEvents = () => {
```

Comment:
usePastEvents og  useEvents er identiske bortsett fra 1 funksjon. Jeg ville heller lagd en generell useEventsGeneral som tar inn en fetch funksjon som første argument. Den blir ikke eksportert

```
     useCacheEventGeneral = (fetchFunction) => {bla bla bla}
     useEvent = useCacheEventsGeneral(getEvents)
     usePastEvents = useCacheEventsGeneral(getPastEvents)
```

Reply:
```useEvents({upcoming:8, done:8})```fetches 8 upcoming and 8 done